lib/IDE/CodeCompletion.cpp

```diff
@@ -2513,6 +2516,54 @@ class CompletionLookup final : public swift::VisibleDeclConsumer {
     return Type();
   }
 
+  void analyzeActorIsolation(
+      const ValueDecl *VD, Type T, bool &implicitlyAsync,
+      Optional<CodeCompletionResult::NotRecommendedReason> &NotRecommended) {
+    auto isolation = getActorIsolation(const_cast<ValueDecl *>(VD));
+
+    switch (isolation.getKind()) {
+    case ActorIsolation::ActorInstance: {
+      if (IsCrossActorReference) {
+        implicitlyAsync = true;
+        // TODO: 'NotRecommended' if this is a r-value reference.
+      }
+      break;
+    }
+    case ActorIsolation::GlobalActor:
+    case ActorIsolation::GlobalActorUnsafe:
+      // TODO: Implement.
+      break;
```

Comment:
Should this be `return` now?

Reply:
I think global actor path also go through `Sendable` analysis when it's implemented.

Reply:
Oh right, the code below still gets skipped due to implicitlyAsync being false.